src/core/animation-loop.js

```diff
@@ -30,6 +27,11 @@ export default class AnimationLoop {
     glOptions = {},
     width = null,
     height = null,
+
+    // options
+    framebuffer = true, // Maintain an offscreen framebuffer of same size as screen
```

Comment:
`framebuffer` is later used as `Framebuffer` object, should this supposed to a boolean flag with a different name?